sim/deathknight/dps/dps_deathknight_test.go

```diff
@@ -12,6 +14,16 @@ func init() {
 	RegisterDpsDeathknight()
 }
 
+func GetAplRotation(dir string, file string) core.RotationCombo {
```

Comment:
It looks like this same function is in every test file, can it be placed in one of the testing files in `core` and shared?

Reply:
good idea, moved it into test_utils